blob/drivertest/drivertest.go

```diff
@@ -50,34 +30,40 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
-// BucketMaker describes functions used to create a bucket for a test.
-// Functions should return the bucket along with a "done" function to be called
-// when the test is complete.
-// If bucket creation fails, functions should t.Fatal to stop the test.
-// See the package comments for more details on required semantics.
-type BucketMaker func(t *testing.T) (*blob.Bucket, func())
+// Harness descibes the functionality test harnesses must provide to run
+// conformance tests.
+type Harness interface {
+	// MakeBucket creates a *blob.Bucket to test.
```

Comment:
You should probably say something about the assumptions about the bucket's contents.

Reply:
Done.